tyr-core/src/main/java/org/jboss/tyr/config/TyrConfiguration.java

```diff
@@ -23,6 +23,9 @@
     @ConfigProperty(name = "github.status.push", defaultValue = "true")
     boolean statusPush();
 
+    @ConfigProperty(name = "github.commit.check.disable", defaultValue = "false")
+    boolean disableCommitCheck();
```

Comment:
better to align this with the property name so 
```suggestion
    boolean commitCheckDisabled();
```